yang/frr-isisd.yang

```diff
@@ -312,6 +322,27 @@ module frr-isisd {
     uses redistribute-attributes;
   }
 
+  grouping leaking-attributes {
```

Comment:
Regardless of the necessity of `route_leaking` container, this is not needed because it's a complete copy of `grouping redistribute-attributes`.